package/src/components/ChannelList/ChannelList.tsx

```diff
@@ -319,6 +336,12 @@ export const ChannelList = (props: ChannelListProps) => {
     sort,
   ]);
 
+  useEffect(() => {
+    if (queryChannelsOverride) {
+      channelManager.setQueryChannelsRequest(queryChannelsOverride);
+    }
+  }, [channelManager, queryChannelsOverride]);
```

Comment:
Does this have to be in useEffect()? Why not put it in the `useMemo` that constructs the `manager`?
This way, the override isn't available in the initial render.

Reply:
It does, because we don't want it to be added only during creation. We can add both I guess if we want to save a render cycle. It's safe to do here of sorts because the actual fetching of channels happens in a `useEffect` as well, meaning we can guarantee this one will run before the one in the hook happens (so it will be up to date).